README.md

```diff
@@ -30,9 +30,6 @@ If you running the Operator as part of the Kubescape cluster components, you nee
 	       "gatewayRestURL": "127.0.0.1:8002",
 	       "kubevulnURL": "127.0.0.1:8081",
 	       "kubescapeURL": "127.0.0.1:8080",
-	       "eventReceiverRestURL": "https://report.armo.cloud",
```

Comment:
You can add here the `serviceDiscovery`

Reply:
`serviceDiscovery` isn't part of `ClusterConfig` so I cannot put it inside, however I can create the file `services.json`

Reply:
I think I should add all 4 files...